aospy/calc.py

```diff
@@ -10,14 +10,18 @@
 import numpy as np
 import xarray as xr
 
-from .constants import Constant, grav
 from . import internal_names
 from . import utils
 from .var import Var
 
 
 logging.basicConfig(level=logging.INFO)
 
+GRAV = (
```

Comment:
Let's rename `GRAV_EARTH` to be more explicit, c.f. my comments on `R_E` and about earth v. other planetary atmospheres.